Lib/sqlite3/dump.py

```diff
@@ -32,15 +32,23 @@ def _iterdump(connection):
         yield('PRAGMA foreign_keys=OFF;')
     yield('BEGIN TRANSACTION;')
 
+    if filter:
+        # Return database objects which match the filter pattern.
+        filter_name_clause = 'AND "name" LIKE %s'
```

Comment:
I guess you meant a placeholder, right?

```suggestion
        filter_name_clause = 'AND "name" LIKE ?'
```

Reply:
Yes, I used Django's style placeholders :smile: , corrected.